jormungandr/src/settings/start/mod.rs

```diff
@@ -72,18 +74,33 @@ impl RawSettings {
             });
         }
 
-        let cmd_level = self.command_line.log_level;
-        let cmd_format = self.command_line.log_format;
+        // If the command line specifies log arguments, check that the arg
+        // is Some(output), else, skip.
         let cmd_output = self.command_line.log_output.clone();
 
-        if cmd_level.is_some() || cmd_format.is_some() || cmd_output.is_some() {
-            entries.push(LogSettingsEntry {
+        if let Some(output) = cmd_output {
+            let cmd_level = self.command_line.log_level;
+            let cmd_format = self.command_line.log_format;
+
+            let command_line_entry = LogSettingsEntry {
                 level: cmd_level.unwrap_or(DEFAULT_FILTER_LEVEL),
                 format: cmd_format.unwrap_or(DEFAULT_LOG_FORMAT),
-                output: cmd_output.unwrap_or(DEFAULT_LOG_OUTPUT),
-            });
+                output,
+            };
+            // log to info! that the output was overriden,
+            // we send this as a message because tracing Subscribers
+            // do not get initiated until after this code runs
+            log_info_msg = Some(format!(
+                "log settings overriden from command line: {:?} replaced with {:?}",
+                entries, command_line_entry
+            ));
```

Comment:
This will leave a message also when no logs are configured in `config.yaml` which is quite OK, so to be perfect, this should be under an `if !entries.is_empty() { ... }` branch.